subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -69,6 +69,8 @@ LOG_MODULE_REGISTER(nrf_cloud_rest, CONFIG_NRF_CLOUD_REST_LOG_LEVEL);
 #define API_LOCATION			"/location"
 #define API_GET_CELL_POS_TEMPLATE	API_VER API_LOCATION "/cell"
 #define API_GET_AGPS_BASE		API_VER API_LOCATION "/agps?"
+#define AGPS_FILTERED			"filtered=true"
```

Comment:
does this need an ampersand in front?

Reply:
It's the first thing after the `API_GET_AGPS_BASE` = '/location/agps?', so an & is not necessary.